### DISCUSSION TITLE
Interactive Editor and Web Config Editor don’t work

### DISCUSSION THREAD

**Discussion by Kai-Denzel-Jane** [Lissy93/dashy#1402] [Help and Support]
Even after rebuilding the application through the dashboard after editing with the dashboards editors or using `yarn build`, if the changes were done through the dashboard they don't seem to save regardless if I press save to disk or not the only way I when really successfully edited my config is manually through the `public/conf.yml`

**Comment by snbzy** [comment]
I'm having the same problem as you, and it would be very convenient if I could modify the config file through the panel
